### ISSUE TITLE
Review physics list

### ISSUE THREAD

**Issue by gipert** [legend-exp/remage#43]
- compare current implementation with latest Geant4 recommendations
- check if angular correlations are correctly enabled

**Comment by EricMEsch** [comment]
Isnt this more or less the same as #180? Also your results comparing the cosmogenics to MaGe should be part of this? @MoritzNeuberger

**Comment by gipert** [comment]
Yes, we have never enabled hadronic physics in the past (MaGe) and it's fine. We want to keep it as is now, i.e. opt-in

**Comment by MoritzNeuberger** [comment]
@EricMEsch, with the issue in the modular physics list regarding neutron captures, one should look into what the current G4-suggested standard is for implementing them and keep remage up to date on that.

**Comment by EricMEsch** [comment]
While i do like the current way of just specifying basic physicslists (like shielding) i agree, that it seems to be more reasonable to just register all processes yourself? So that you can keep better track of what is happening.

Like we should pick out a set of processes for default remage (and then we can argue about which options we want to give to the user)

**Comment by gipert** [comment]
yes i agree that here we should be verbose and include/exclude processes in detail, instead of relying on some "bundled" G4 method which could expose us to unexpected changes depending on the G4 version

**Comment by ManuelHu** [comment]
but we also need to take into account that these constructors are the "official" way of constructing physics lists in Geant4, so we would need to follow the upstream changes much more closely.

**Comment by MoritzNeuberger** [comment]
I agree with Manuel, also considering that we can only really compare our results with other experiments/simulation stacks when we use the same models. And the usual way people quote what they did is by stating the default physics list they used.

**Comment by gipert** [comment]
This could be also closed?
